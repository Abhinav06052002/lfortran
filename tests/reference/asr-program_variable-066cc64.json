{
    "basename": "asr-program_variable-066cc64",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/program_variable.f90",
    "infile_hash": "a94dee5e4e732105f867058d4cf7440bd8639fea0a1f7cae424caf30",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-program_variable-066cc64.stderr",
    "stderr_hash": "c63ac7db2dd11f7aac268c08904dead16ac65e98fb2aa5695cf273a7",
    "returncode": 2
}