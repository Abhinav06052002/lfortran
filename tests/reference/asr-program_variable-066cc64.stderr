semantic error: Symbol 'foo' must be a variable or a function
 --> tests/errors/program_variable.f90:3:7
  |
3 | bar = foo
  |       ^^^ 'foo' is a 'Program', not a variable or a function
